reg_tests/test_files/ActLineSimpleNGP/ActLineSimpleNGP.i

```diff
@@ -40,17 +40,17 @@ linear_solvers:
   type: tpetra
   method: gmres
   preconditioner: sgs
-  tolerance: 1e-5
-  max_iterations: 50
+  tolerance: 1e-12
```

Comment:
What is the timing difference with these solver tolerances?  I'm not sure we want to modify the regression test with these tight of tolerances since it will affect nightly/CI times.

Reply:
@psakievich For NGP tests that run on both CPU and GPU, we use tight tolerances in linear solvers so that we can still use CPU golds to check the GPU runs to a tolerance tighter than `1e-8`. So this change is appropriate and is being done for all other GPU tests.

Reply:
@sayerhs okay thanks for the clarification